src/proxy-page/steps/addLibrariesJS.ts

```diff
@@ -42,7 +42,8 @@ export default (): Step => {
           konviwSpaceKey: "${context.getSpaceKey()}",
           konviwPageId: "${context.getPageId()}",
           konviwTitle: "${context.getTitle()}",
-          konviwExcerpt: "${context.getExcerpt()}"
+          konviwExcerpt: "${context.getExcerpt()}",
+          konviwVersion: "${context.getVersion()}",
```

Comment:
Not sure this will work passing an object JSON (Version) as iFrameResizer will stringify and we will have to parse back to an object in the client.

Reply:
it works but it's not optimal as solution, so i will stringify  the version